packages/http/src/forwarder/__tests__/HttpForwarder.spec.ts

```diff
@@ -7,20 +7,21 @@ describe('HttpForwarder', () => {
   const forwarder = new HttpForwarder();
 
   beforeEach(() => {
-    jest.spyOn(axios, 'default').mockImplementation(jest.fn);
+    jest.spyOn(axios, 'default');
```

Comment:
The intention or the `mockImplementation` was to actually stub that function not to spy and pass through it. Why have you decided to change this?

Reply:
1. If your intention is only to check if a function was called, you need a `spy` and not a `mock`. I understand that touching the network is a PITA; the function gets mocked anyway below, so there's no reason to mock it in a before each (which can actually be completely removed)

2. This code was wrong anyway — the correct call should have been
```diff
- jest.spyOn(axios, 'default').mockImplementation(jest.fn);
+ jest.spyOn(axios, 'default').mockImplementation(jest.fn());
```

Reply:
> 1. If your intention is only to check if a function was called, you need a `spy` and not a `mock`. I understand that touching the network is a PITA; the function gets mocked anyway below, so there's no reason to mock it in a before each (which can actually be completely removed)

The reason it was in the before each is to avoid problems with new tests (e.g. someone adds a test and don't realize they use an actual network). Better be safe than sorry ;)

> 2. This code was wrong anyway — the correct call should have been
> 
> ```diff
> - jest.spyOn(axios, 'default').mockImplementation(jest.fn);
> + jest.spyOn(axios, 'default').mockImplementation(jest.fn());
> ```

:+1: